packages/cli/src/commands/run.ts

```
@@ -174,6 +205,18 @@ export async function runRun(workflow: string, options: Record<string, unknown>)
     }
   }
 
+  const previewWorkflow = expandedWorkflow as
+    | { steps?: Array<{ name: string; input?: Record<string, unknown>; output?: { path?: string; schema?: string } }> }
```

ESLint

[error] Replace `·steps?:·Array<{·name:·string;·input?:·Record<string,·unknown>;·output?:·{·path?:·string;·schema?:·string·}·}>` with `⏎········steps?:·Array<{⏎··········name:·string;⏎··········input?:·Record<string,·unknown>;⏎··········output?:·{·path?:·string;·schema?:·string·};⏎········}>;⏎·····`

(prettier/prettier)